modelling/src/main/java/org/axonframework/modelling/command/AggregateAnnotationCommandHandler.java

```diff
@@ -429,15 +443,7 @@ public Object handle(CommandMessage<?> command) throws Exception {
             String commandMessageAggregateId = commandMessageVersionedId.getIdentifier();
 
             Aggregate<T> instance = repository.loadOrCreate(commandMessageAggregateId, factoryMethod);
-            Object commandResult = instance.handle(command);
```

Comment:
Shouldn't this change only reside in pull request #2191?

Reply:
You're right. I've amended the commit to remove these changes.